projects/focil-geth-and-prysm-poc.md

```diff
@@ -0,0 +1,80 @@
+# Fork-Choice Enforced Inclusion Lists (FOCIL) Proof of Concept in Geth and Prysm
+
+Enhancing Decentralization and Censorship Resistance in Block Production
+
+## Motivation
+
+**Centralization Issue:**
+- Block building rights are auctioned to specialized entities known as builders.
+- Over the past year, a few sophisticated builders have dominated block production due to economies of scale.
+
+**Censorship Resistance:**
+- Concentrated block production weakens the network’s censorship resistance properties.
+- 60% of builders filter transactions from sanctioned addresses, while 90% of validators do not engage in censorship.
+
+## Project Description
+
+A randomly selected set of validators forms an inclusion list ($IL$) committee to create and broadcast local inclusion lists. The block producer aggregates these lists into a canonical aggregate ($IL_{agg}$), which is included in the block and validated by attesters to ensure block validity.
+
+## Specification
+
+**Consensus Layer:**
+
+- **New `InclusionList` object:**
+  - `arr InclusionList`: An array of $IL$ from every member of the $IL$ committee.
+  - `AggregateInclusionLists` function: Aggregates local $IL$ and creates $ IL_{agg} $.
+  
+- **EvaluateInclusionList function:**
+
+   [![\\ $$ \\ \text{Eval}(IL_{\text{agg}}^{\text{attester}}, IL_{\text{agg}}^{\text{proposer}}, \Delta) =  \\ \begin{cases}  \\ \text{True} & \text{if } \frac{|IL_{\text{agg}}^{\text{attester}} \cap IL_{\text{agg}}^{\text{proposer}}|}{|IL_{\text{agg}}^{\text{attester}}|} \geq \Delta \\  \\ \text{False} & \text{otherwise}  \\ \end{cases} \\ ](https://latex.codecogs.com/svg.latex?%5C%5C%20%24%24%20%5C%5C%20%5Ctext%7BEval%7D(IL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Battester%7D%7D%2C%20IL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Bproposer%7D%7D%2C%20%5CDelta)%20%3D%20%20%5C%5C%20%5Cbegin%7Bcases%7D%20%20%5C%5C%20%5Ctext%7BTrue%7D%20%26%20%5Ctext%7Bif%20%7D%20%5Cfrac%7B%7CIL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Battester%7D%7D%20%5Ccap%20IL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Bproposer%7D%7D%7C%7D%7B%7CIL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Battester%7D%7D%7C%7D%20%5Cgeq%20%5CDelta%20%5C%5C%20%20%5C%5C%20%5Ctext%7BFalse%7D%20%26%20%5Ctext%7Botherwise%7D%20%20%5C%5C%20%5Cend%7Bcases%7D%20%5C%5C%20)](#_)
```

Comment:
This is not rendering correctly 
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions